mod/01_rating/set/type/metric_value.rb

```diff
@@ -106,31 +106,40 @@ def create_source
 end
 
 format :html do
-  def default_new_args args
+  def get_structure args
+    if args[:company]
+      'metric company add value'
+    elsif args[:source]
+      'metric_source_add_value'
+    elsif args[:metric]
+      'metric add value'
+    else
+      'default add metric value'
+    end
+  end
+
+  def set_hidden_args args
     if !args[:source]
+      view = (args[:metric] || args[:company]) ? :titled : :open
```

Comment:
the redirection should a normal open view if the metric value is created on a new default metric value form.